client/ratings_reviews/components/ReviewTile.jsx

```diff
@@ -1,9 +1,42 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
 
-const ReviewTile = (props) => (
-  <div>
-    reviewtile
-  </div>
-);
+class ReviewTile extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      rating: 0
+    };
+    this.updateStars = this.updateStars.bind(this);
+  }
+
+  updateStars() {
+    var percentage = (this.props.review.rating / 5) * 100;
+    var percentageRounded = (Math.round(percentage * 4) / 4).toFixed(2);
+  }
+
+  render() {
+    var responseVisibility = true;
+    if (this.props.review.response === null || this.props.review.response.length === 0) {
+      responseVisibility = false;
+    }
+    return (
+      <div class='review-tile'>
+        <div id={this.props.review.review_id}>Rating: {this.props.review.rating}
```

Comment:
Curious about the choice of id for the div here, would you use that to target via CSS rules?